utils/gen_onnx_mlir.py

```diff
@@ -41,6 +41,13 @@
 
 import onnx
 
+current_onnx_version = "1.9.0"
```

Comment:
Super useful. I would add a comment "change this variable only when upgrading the ONNX support within ONNX-MLIR"